assets/sass/modules/_btns.scss

```diff
@@ -226,6 +226,24 @@ $od-secondary-button-action-text-color: #124a94;
   }
 }
 
+@mixin social-auth-button-colorless-template($bg-image) {
+  background-color: $other-label-bg-color;
+  background: $bg-image no-repeat 12px/24px,
```

Comment:
curious, why the `12px/24px` vs `10px` used in `social-auth-button-template`

Reply:
There are two current approaches to the icon positioning and sizing:

- Facebook, LinkedIn, and Microsoft display 30x30px icons at 1x size shifted over 10px (the `10px` you mention)
- Apple and Google display 20x20px icons at 1x size shifted over 15px (overriding the `10px` you mention)

For the new icons, we wanted to display them at 0.5x size because it looks better on a retina display. We picked 24px wide [somewhat arbitrarily](https://oktainc.atlassian.net/browse/OKTA-422990), and packaged 48px icons. The resulting approach is:

- Our new IdPs display 48x48px icons at 0.5x size (the `24px` in your message) shifted over 12px (the `12px` in your message)